index.html

```diff
@@ -7,7 +7,7 @@
   <h1 class="page-heading">{{ site.title }}</h1>
 
   <select class="js-canton-selector canton-selector">
-    <option>Select your canton</option>
+    <option>Ingresá tu cantón</option>
```

Comment:
This needs to read "Elige tu cantón" Ingresar means "admit" or similar...like admit to a hospital